projects/kit/components/input-number/input-number.directive.ts

```diff
@@ -86,13 +86,17 @@ export class TuiInputNumberDirective extends TuiControl<number | null> {
     protected readonly options = inject(TUI_INPUT_NUMBER_OPTIONS);
     protected readonly element = tuiInjectElement<HTMLInputElement>();
 
-    protected readonly inputMode = computed(() => {
-        if (this.isIOS && this.min() < 0) {
+    protected readonly inputMode = computed((needMinus = this.min() < 0) => {
+        if (this.isIOS && needMinus) {
```

Comment:
&& this.min() < 0, since it's used only once. And we can drop all the { return ... } now since it's just a ternary